bin/zkEnv.cmd

```diff
@@ -46,5 +46,8 @@ if not exist "%JAVA_HOME%"\bin\java.exe (
   goto :eof
 )
 
+REM strip off trailing \ from JAVA_HOME or java does not start
+if "%JAVA_HOME:~-1%" EQU "\" set "JAVA_HOME=%JAVA_HOME:~0,-1%"
+ 
 set JAVA="%JAVA_HOME%"\bin\java
 
```

Comment:
??? this change has already existed in the master

Reply:
apologies yes i had not sync'ed my fork. Would you like me to create a different PR or is ok ?

Reply:
Although it has no code conflict with the master branch, it will not apply correctly when wanting to merge into master.so you need to revert this commit

Reply:
i reverted the obsolete commit from the PR